src/Microsoft.Docs.Template/Models/LandingData.cs

```diff
@@ -109,7 +109,6 @@ public sealed class LandingDataItem
 
         public LandingDataRow[] Rows { get; set; }
 
-        [Html]
```

Comment:
For html attribute, it also do something else, can we just remove the dependency resolving from these logics:

```
            if (attribute is HtmlAttribute)
            {
                var html = HtmlUtility.TransformLinks((string)value, href =>
                {
                    var (error, link, _) = context.DependencyResolver.ResolveLink(href, file, file, buildChild);
                    context.Report.Write(file.ToString(), error);
                    return link;
                });
                return HtmlUtility.StripTags(HtmlUtility.LoadHtml(html)).OuterHtml;
            }
```